### PR TITLE
Feature/kx 114 update connection in document page frontend

### PR DESCRIPTION
**Pull request by GiuseppeArbore**

### CODE REVIEW

File: client/src/API.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { mongoose } from '@typegoose/typegoose';
-import { KxDocument, PageRange, Scale } from './model';
+import { KxDocument, PageRange, Scale, Connections } from './model';
```
Comment (golnazsaraji) [inline client/src/API.ts:2]: this is the scenario I checked: 
for each connection, I added two documents. On details, whenever I remove one document, the error is: 
Screen Shot 2024-12-15 at 15 20 20

Screen Shot 2024-12-15 at 15 23 30
just consider, I created the document before that to be sure about the new structure of the document

**Review by golnazsaraji — CHANGES_REQUESTED** [review]

**Review by AlbertoPedalino — CHANGES_REQUESTED** [review]
image
unable to see the document page

**Review by AlbertoPedalino — CHANGES_REQUESTED** [review]
check line 278 in document.tsx
current : {scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale.type}
suggestion:  {scale && scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale?.type || 'Unknown Scale'} .

File: client/src/API.ts (hunk 1 above)
Comment (GiuseppeArbore) [reply client/src/API.ts:2]: There isn't the backend. This is frontend + UI... The backend is another branch

**Comment by GiuseppeArbore** [comment]
> check line 278 in document.tsx current : {scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale.type} suggestion: {scale && scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale?.type || 'Unknown Scale'} .

Scale is mandatory, so there is no way that it could be null

> check line 278 in document.tsx current : {scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale.type} suggestion: {scale && scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale?.type || 'Unknown Scale'} .

scale is mandatory and there is no way in which it could be undefined

**Review by AlbertoPedalino — APPROVED** [review]

**Pull request merged**